src/settings-ui/Settings.UI.Library/PowerPreviewProperties.cs

```diff
@@ -80,6 +80,23 @@ public bool EnableMonacoPreview
             }
         }
 
+        private bool enableMonacoPreviewWordWrap = true;
```

Comment:
I don't think enable prefix is needed here. `MoncaoPreviewWordWrap` should be enough